src/Swashbuckle.AspNetCore.Cli/Program.cs

```diff
@@ -15,18 +15,26 @@ namespace Swashbuckle.AspNetCore.Cli
 {
```

Comment:
I'm curious to know what the rationale is for the changes to this file - seems like it's just a subjective preference to have the logic in instance methods vs static Main or am I missing something?

Reply:
I was following the pattern we have in the aspnetcore repo for some tools, can change to make Main public and call that instead if it's preferred.

Reply:
Yeh - if the only material benefit is consistency with an external repo, I'd prefer to revert to how it was. Thanks.